examples/edi-to-bal/order1.edi

```diff
@@ -0,0 +1,6 @@
+HDR*ORDER_1201*ABC_Store*2008-01-01~
```

Comment:
Instead of `order1.edi` shall we rename it to `order.edi` and update the .md descriptions as well.